### ISSUE TITLE
Menus in the wrong place

### ISSUE THREAD

**Issue by johnshaughnessy** [mozilla/hubs#2420]
**Description**
Menus not appearing where they should

**To Reproduce**
I don't know how to reproduce it!
https://poly.google.com/view/4w-_VbcSECP

**Expected behavior**
Menus should be in front of the model, facing you.

**Screenshots**
![image](https://user-images.githubusercontent.com/4072106/80247169-73523980-8622-11ea-8374-427a32cee041.png)
![image](https://user-images.githubusercontent.com/4072106/80247214-86fda000-8622-11ea-8b49-862fc071cd6d.png)

**Hardware** 
Tested on Linux, Chrome

**Comment by johnshaughnessy** [comment]
Elgin is seeing these issues on Pico headsets, too. 

1) camera controls (like the one for muting/unmuting) are inside of the camera object, so not accessible
2) the other one was z-fighting of the video controls, but doesnt seem to repro

**Comment by johnshaughnessy** [comment]
Tried hard to reproduce this on my own today and couldn't figure out how. Hoping to see this again on Friday and be able to debug on hubs.mozilla.com, but for now un-assigning myself because I haven't found a way to reproduce it. 

I was hoping that this would have explained and fixed the issue: https://github.com/mozilla/hubs/pull/2422

But we saw this during our Wednesday standup (after the PR above had been deployed). Interestingly, when it failed, I believe it failed for multiple people (possibly everyone, but I forgot to ask). I am not sure what to make of that.
